Library/Homebrew/test/test_tab.rb

```diff
@@ -75,6 +76,7 @@ def test_from_file
     refute_predicate tab, :built_as_bottle
     assert_predicate tab, :poured_from_bottle
     assert_equal "Homebrew/homebrew", tab.tap
+    assert_equal :stable, tab.spec
```

Comment:
Would be good to have a test without a `tab.spec` too just to verify that case works properly as well (for old tabs)